bot/handlers/commands.py

```diff
@@ -1,12 +1,16 @@
-from aiogram import Router, F
+from datetime import datetime
+
+from aiogram import Router
 from aiogram.filters import Command
 from aiogram.types import Message, InlineKeyboardButton
 from aiogram.utils.keyboard import InlineKeyboardBuilder
 
+from ping_app.periods_service import PeriodService
 from settings_reader import config
 
 
 router = Router()
+ps = PeriodService()
```

Comment:
Is there any reason to "save" some symbols here? Like I would name it `perios_service` so it would be easier to read when this file becomes more than 20 lines.